src/server/services/UrlManagementService.ts

```diff
@@ -48,7 +48,8 @@ export class UrlManagementService implements UrlManagementServiceInterface {
     const owner = await this.userRepository.findUserByUrl(shortUrl)
     if (owner) {
       throw new AlreadyExistsError(
-        `Short link "${shortUrl}" is owned by ${owner.email}`,
+        `Short link "${shortUrl}" is used. Click here to find out more`,
+        // `Short link "${shortUrl}" is owned by ${owner.email}`,
```

Comment:
Is this commented out code meant to be kept in the codebase?

Reply:
yes, as a reference if we decide to revert the functionality back in the future as the alert was originally meant to display the owner of the link

Reply:
Just remove the comment - we can fit blame if we want to reverse the change